server/ansible/pbench-server-disable.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: disable pbench-server
+  hosts: servers
+  remote_user: root
```

Comment:
Shouldn't this be doable only using the `pbench` user?

Reply:
Yes. It does require a separate ssh-copy-id by the admin, but the playbooks should use the minimal privilege possible.